src/bundle/Resources/public/js/scripts/sidebar/btn/content.edit.js

```diff
@@ -0,0 +1,21 @@
+(function () {
+    const editButton = document.querySelector('.ez-btn--edit[data-actions="edit"]');
+
+    editButton.addEventListener('click', () => {
+        const languageButtons = document.querySelectorAll('.ez-extra-actions--edit[data-actions="edit"] .form-check [type="radio"]');
+
+        if (languageButtons.length !== 1) {
+            return;
+        }
+
+        const triggerOnClickEventForLanguage = (button) => {
+            button.checked = true;
+            const onChangeEvent = document.createEvent("HTMLEvents");
+            onChangeEvent.initEvent("change", true, false);
+            button.dispatchEvent(onChangeEvent);
```

Comment:
It can be simplified, to just `button.dispatchEvent(new CustomEvent('change'))`.
Another question, why do you need that event?

Reply:
There is an event listening for this form submission (checking for draft conflicts).